scripts/build.ps1

```diff
@@ -54,6 +54,8 @@ Param(
     [String[]] $Steps = @("InstallDotnet", "Restore", "UpdateLocalization", "Build", "Publish", "PrepareAcceptanceTests")
 )
 
+$ErrorActionPreference = 'Stop'
```

Comment:
Would there be any use case in which we'd like to have this error action preference configurable ?

Reply:
Nope. This is a best practice. The common.lib.ps1 file actually sets this as well. Setting this to Stop, and optionally overriding it to Continue on commands that are allowed to fail is how this is done.